contracts/v-testnet-launch/GrtLiquidityWallet.sol

```diff
@@ -54,10 +57,10 @@ contract GrtLiquidityWallet is OwnableUpgradeable, UUPSUpgradeable {
         address to,
```

Comment:
@tcoratger do you think it's valid to check if the address is zero for `token` and `to` params?